index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>mobile version</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="header">
+        <a class="logo">My logo</a>
+        <button type="button" class="btn">
+          <img alt="nav" class="menu" src="./images/Union.png" />
+        </button>
+      </nav>
+    </header>
+    <section class="flex-container">
+      <div class="text">
+        <h1>I'm Loic<br />Glad to see<br />you</h1>
```

Comment:
- [ ] The `Glad to see you` part is a sub-title. Thus, you should put in its own element and let's use `<h2> .... <h6>` for semantic reasons. The ` I'm Loic` part should remain the `<h1>`, however.
- [ ] You should use `40px` font-size for this heading. Both the main title and the subtitle. You are currently using `1.5em`, and if the default is 16px it would be around 24px.